problems/countNumbers.js

```diff
@@ -8,10 +8,21 @@
  * ex: countNumbers([1,1,1,2,2,3,4]) 
  * returns { 1:3, 2:2, 3:1, 4:1 }
  */
-
-function countNumbers() {
- 
+//Object.keys(array).length:
+const countNumbers = (arr) => {
+    let newArr = arr
+    let countObj = {}
+    newArr.map((el) => {
```

Comment:
We should be using forEach here instead of map. B/c we're updating an outside variable countObj we're using our callback function for its side effects. We want to use map when we're going to be using the array that it automatically returns.